.github/workflows/integration_tests.yml

```
@@ -147,17 +146,17 @@ jobs:
         if: (success() || failure()) && (inputs.target == 'iqm' || github.event_name == 'schedule' || inputs.target == 'nightly')
         # The following tests are disabled because the demo server only supports
         # 5 qubits and the test tries to use more.
-        # - test/NVQPP/integration/graph_coloring-1.cpp
-        # - test/NVQPP/integration/graph_coloring.cpp
+        # - test/NVQPP//graph_coloring-1.cpp
+        # - test/NVQPP//graph_coloring.cpp
```

There are a couple of items like this.
```suggestion
        # - test/NVQPP/graph_coloring-1.cpp
        # - test/NVQPP/graph_coloring.cpp
```